Docs/diataxis 1 (#888)
iotaledger/wasp: documentation/docs/overview.md

---
description: IOTA Smart Contracts allow you to run smart contracts on top of the IOTA Tangle.
image: /img/logo/WASP_logo_dark.png
keywords:
- Smart Contracts
- Core Concepts
- scaling
- flexibility
- explanation
---

# IOTA Smart Contracts

The IOTA Smart Contracts is a protocol that brings scalable and flexible smart contracts into the IOTA ecosystem. It allows anyone to spin up a smart contract blockchain and anchor it to the IOTA Tangle, a design that offers a multiple advantages.

## ISC Advantages

### Scaling and Fees

Due to the ordered structure and execution time of a smart contract, a single blockchain can only handle so many transactions per second before it needs to decide on which transactions it needs to postpone until other blocks are produced, as there is no processing power available for them on that chain. This eventually results in high fees on many chains, and limited functionality. 

> As ISC allows many chains to be anchored to the IOTA Tangle and lets them communicate with each other, we can simply add additional chains once this becomes a problem. This results in lower fees over solutions that just use a single blockchain for all their smart contracts.

Reviewer: ```suggestion
As ISC allows many chains to be anchored to the IOTA Tangle and lets them communicate, we can add additional chains once this becomes a problem. This results in lower fees over solutions that use a single blockchain for all their smart contracts.
```

### Custom Chains

> Given that anyone can start a new chain and set its rules, a lot is possible. Not only do you have full control over how the fees are handled on the chain you set up, but you also have full control over the validators and access to your chain. You can even spin up a private blockchain with no public data besides the state hash that is anchored to the main IOTA Tangle. This allows parties that need private blockchains to use the security of the public IOTA network without actually exposing their blockchain to the public.

Reviewer: ```suggestion
Given that anyone can start a new chain and set its rules, a lot is possible. You have complete control over how the fees are handled on the chain you set up, but you also have full control over the validators and access to your chain. You can even spin up a private blockchain with no public data besides the state hash anchored to the main IOTA Tangle. This allows parties that need private blockchains to use the security of the public IOTA network without actually exposing their blockchain to the public.
```
